build.sbt

```diff
@@ -258,6 +258,7 @@ lazy val docs = project
         "canonical.base_url" -> "https://doc.akka.io/docs/akka-management/current",
         "scala.binary_version" -> scalaBinaryVersion.value,
```

Comment:
This needs to become `scala.binary.version`

Reply:
Strange, it actually works with `scala.binary.version` in the dependency snippets without changing this. Anyway, I changed all usages of `scala.binary.version` to `"scala.binary_version"` since that was what used previously.

Reply:
Hm, I need to check why that worked...

Reply:
Ah, it is the new feature in the Paradox that requires that... I'll fix once this is merged.